src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      autocomplete="on"
+    >
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <label class="block">Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        
+        <label class="block">Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        
+        <label class="block">How old are You?
+          <input
+            type="number"
+            name="arenum"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="block">Full date of birth:
+          <input
+            type="date"
+            name="dbirth"
+            required
+          >
+        </label>
+
+        <label class="block">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>Registration</legend>
+        <label class="block">E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+            multiple
+          >
+        </label>
+        
+        <label>Password:
+          <input
+            type="password"
+            name="pass"
+            required
+            autocomplete="off"
+            min="6"
+            max="20"
+          >
+        </label>
+      </fieldset>
+   
+      <fieldset class="section">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="block">Do you love cats?
+          <input
+            type="radio"
+            name="c"
+          > Yes
+
+          <input
+            type="radio"
+            name="c"
+          > No
```

Comment:
Wrap each radio with label to make `yes` and `no` words clickable 
```suggestion
          <input
            type="radio"
            name="c"
          > 
          Yes

          <input
            type="radio"
            name="c"
          > 
          No
```